src/components/Calendar/CalendarPaneToolbar.js

```diff
@@ -55,6 +56,7 @@ const CalendarPaneToolbar = (props) => {
     const isMobileScreen = useMediaQuery('(max-width:630px)');
 
     const [anchorEl, setAnchorEl] = React.useState(null);
+    const [openSchedules, setOpenSchedules] = React.useState(false);
```

Comment:
Nit: we should just import useState at the top
```jsx
import React, { useState } from 'react';
```

So that this will simply become:
```suggestion
    const [anchorEl, setAnchorEl] = useState(null);
    const [openSchedules, setOpenSchedules] = useState(false);
```